monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -68,7 +63,7 @@
         """
         filename = secure_filename(request_.files["filepond"].filename)
         file_path = (
-            Path(env_singleton.env.get_config().data_dir_abs_path).joinpath(filename).absolute()
+            Path(PostBreachFilesService.get_custom_pba_directory()).joinpath(filename).absolute()
         )
```

Comment:
Why do we need these parentheses?

Reply:
The line is longer that 100 characters, so this is how black formatted it.